django/db/backends/base/features.py

```diff
@@ -287,9 +287,8 @@ class BaseDatabaseFeatures:
     create_test_procedure_without_params_sql = None
     create_test_procedure_with_int_param_sql = None
 
-    # SQL to create a table with a composite primary key for use by the Django
-    # test suite.
-    create_test_table_with_composite_primary_key = None
+    # Does the backend supports composite primary key?
+    supports_composite_pk = True
```

Comment:
If we add this flag, I suppose we want to add `required_db_features = {"supports_composite_pk"}` to all our composite primary key models? Do we need this?

Reply:
Maybe we don't need it yet so I'll just remove it. We usually feature gate new ORM features but I guess we can wait until it's requested by a third party backend maintainer in this case (e.g. I'm pretty sure Mongo doesn't support it)